.release-please-manifest.json

```diff
@@ -1,4 +1,5 @@
 {
+	".github/actions/happy-cleanup": "1.0.0",
```

Comment:
I don't think you need this, having it here might make it bump to v1.1.0 to start 🤷